server/webapp/localize_ja.xml

```diff
@@ -780,6 +780,7 @@
     <entry key="SCM_USED_IN_PIPELINES_LINK">Show pipelines using this SCM</entry>
     <entry key="STALE_PIPELINE_CONFIG">Someone has modified the configuration for pipeline ''{0}''. Please update your copy of the config with the changes.</entry>
     <entry key="PIPELINE_CONFIG_VALIDATION_FAILED">Validations failed for pipeline ''{0}''. Please correct and resubmit.</entry>
```

Comment:
Duplication? Can `PIPELINE_CONFIG_VALIDATION_FAILED` be implemented via `ENTITY_CONFIG_VALIDATION_FAILED` ?